test/testCSVParser.cc

```diff
@@ -129,6 +129,36 @@ SUITE(CSVParser)
       
       CHECK(osn.str().find("missing numerical data") != std::string::npos);
     }    
+
+  TEST_FIXTURE(CSVDialog,loadWebFile)
+    {
+	  spec=DataSpec();	
+      string url="https://www.dickimaw-books.com/latex/admin/html/examples/ordergroups.csv";
```

Comment:
This test is too dependent on somebody else's (unrelated) website. It would probably be sufficient to download that csv file, and then convert the URL into a file:// URL. Also please check before doing so that we have a clear license to use the file that way - maybe just create a random .csv file to avoid any copyright concerns.

Reply:
I've found several online random CSV generators, but those I've tried don't give me a URL download link. For example, https://www.generatedata.com/

So, at first I decided to go the route you followed with Ravel, i.e., dowloading the daily COVID reports from github, but then it occurred to me we might have uploaded a CSV file at sourceforge under one of the raised tickets. Voila: https://sourceforge.net/p/minsky/ravel/20/

So, I'm using BIS_GDP.csv now.